docs/docker.md

```diff
@@ -17,325 +17,96 @@ Once you have Docker installed, simply prepare the config file (e.g. `config.jso
+    curl https://raw.githubusercontent.com/freqtrade/freqtrade/master_pi/docker-compose.yml -o docker-compose.yml
 
-!!! Note
+    # Pull the freqtrade image
+    docker-compose pull
+
+    # Create user directory structure
+    docker-compose run --rm freqtrade create-userdir --userdir user_data
+
+    # Create configuration - Requires answering interactive questions
+    docker-compose run --rm freqtrade new-config --config user_data/config.json
+    ```
+
+The above snippet creates a new directory called `ft_userdata`, downloads the latest compose file and pulls the freqtrade image.
+The last 2 steps in the snippet create the directory with `user_data`, as well as (interactively) the default configuration based on your selections.
+
+!!! Question "How to edit the bot configuration?"
     You can edit the configuration at any time, which is available as `user_data/config.json` (within the directory `ft_userdata`) when using the above configuration.
 
-#### Adding your strategy
+#### Adding a custom strategy
+
+1. The configuration is now available as `user_data/config.json`
+2. Copy a custom strategy to the directory `user_data/strategies/`
+3. add the Strategy' class name to the `docker-compose.yml` file
 
-The configuration is now available as `user_data/config.json`.
-You should now copy your strategy to `user_data/strategies/` - and add the Strategy class name to the `docker-compose.yml` file, replacing `SampleStrategy`. If you wish to run the bot with the SampleStrategy, just leave it as it is.
+The `SampleStrategy` is run by default.
 
-!!! Warning
+!!! Warning "`SampleStrategy` is just a demo!"
     The `SampleStrategy` is there for your reference and give you ideas for your own strategy.
     Please always backtest the strategy and use dry-run for some time before risking real money!
 
 Once this is done, you're ready to launch the bot in trading mode (Dry-run or Live-trading, depending on your answer to the corresponding question you made above).
 
-``` bash
-docker-compose up -d
-```
+=== "Docker Compose"
+    ``` bash
+    docker-compose up -d
+    ```
 
 #### Docker-compose logs
 
-Logs will be written to `user_data/logs/freqtrade.log`. 
-Alternatively, you can check the latest logs using `docker-compose logs -f`.
+Logs will be located at: `user_data/logs/freqtrade.log`. 
+You can check the latest log with the command `docker-compose logs -f`.
 
 #### Database
 
-The database will be in the user_data directory as well, and will be called `user_data/tradesv3.sqlite`.
+The database will be at: `user_data/tradesv3.sqlite`
 
 #### Updating freqtrade with docker-compose
 
-To update freqtrade when using docker-compose is as simple as running the following 2 commands:
+To update freqtrade when using `docker-compose` is as simple as running the following 2 commands:
 
-``` bash
-# Download the latest image
-docker-compose pull
-# Restart the image
-docker-compose up -d
-```
+=== "Docker Compose"
```

Comment:
I'm still indecisive if i like this or not ... it looks good on the one hand - but the commands it contains are really just shell commands, not compose commands.

Reply:
I have removed the label on this.